platform-core-module/src/test/java/com/softicar/platform/core/module/user/configuration/table/UserSpecificTableConfigurationPersistenceApiTestWithQuery.java

```diff
@@ -0,0 +1,96 @@
+package com.softicar.platform.core.module.user.configuration.table;
+
+import com.softicar.platform.common.core.interfaces.IStaticObject;
+import com.softicar.platform.db.runtime.query.IDbQuery;
+import com.softicar.platform.emf.data.table.EmfDataTableDivBuilder;
+import com.softicar.platform.emf.data.table.EmfDataTableDivMarker;
+import org.junit.Test;
+
+/**
+ * Test for {@link UserSpecificTableConfigurationPersistenceApi} on
+ * {@link IDbQuery}.
+ *
+ * @author Oliver Richers
+ */
+public class UserSpecificTableConfigurationPersistenceApiTestWithQuery extends AbstractUserSpecificTableConfigurationPersistenceApiTest {
```

Comment:
This might be unprecedenced in our code base: I think **all** of our other test (:= class which contain at least one `@Test` method) classes have a `Test` suffix. I think this class should be renamed, even if the resulting name might appear artificial.